Revert colors back to 0.6.2

paladox opened the pull request:
It was causing colors to not show in grunt.

@shama or @vladikoff please could you review and merge.

paladox commented:
I doint know why this is failing since I'm only going back to a version that was working before.

shama commented:
Tracking https://github.com/Marak/colors.js/pull/154 for a fix.

paladox commented:
It's unlikely it will be merged since there hasent been a commit since September 2015.

I've submitted the pull here https://github.com/gruntjs/grunt-legacy-log/pull/23 that it uses that commit

vladikoff commented:
Appveyor issues: 

```
Running "nodeunit:util" (nodeunit) task
Testing index.js.............F.................
>> Log instance - writeflags
>> Error: 'Flags: a, b\n' == 'Flags: \u001b[36ma\u001b[39m, \u001b[36mb\u001b[39m\n'
>> at stdoutEqual (test\index.js:26:8)
>> at Object.exports.Log instance.writeflags (test\index.js:210:5)
>> at Object.exports.Log instance.setUp (test\index.js:57:5)
 
>> Log instance - writeflags
>> Error: 'Prefix: a, b\n' == 'Prefix: \u001b[36ma\u001b[39m, \u001b[36mb\u001b[39m\n'
>> at stdoutEqual (test\index.js:26:8)
>> at Object.exports.Log instance.writeflags (test\index.js:213:5)
>> at Object.exports.Log instance.setUp (test\index.js:57:5)
 
>> Log instance - writeflags
>> Error: 'Prefix: a, b=false, c=0, d=null\n' == 'Prefix: \u001b[36ma\u001b[39m, \u001b[36mb=false\u001b[39m, \u001b[36mc=0\u001b[39m, \u001b[36md=null\u001b[39m\n'
>> at stdoutEqual (test\index.js:26:8)
>> at Object.exports.Log instance.writeflags (test\index.js:216:5)
>> at Object.exports.Log instance.setUp (test\index.js:57:5)
 
Warning: 3/209 assertions failed (125ms) Use --force to continue.

```

paladox commented:
@vladikoff yep that's because we updated to colors 1.1.2 which broke some programes such as ones based on Linux like git for windows and cygwind. Plus powershell I doint think works with colors 1.1.2 because of the new check that was added that broke it meaning it wasn't really testing colors because it would switch colors off because of the check. But https://github.com/Marak/colors.js/pull/154 should fix it but is unlikely to be merged any time soon looking at the commits the last one was in septemeber 2015.

But appveyor was added after we updated colors to 1.1.2 so if we added it when we were using 0.6.2 it would likely have been broken.
